doc/source/release/1.11.0-notes.rst

```diff
@@ -96,7 +96,7 @@ type is preferred, similar to the ``datetime.datetime`` type in the Python
 standard library. Accordingly, datetime64 no longer assumes that input is in
 local time, nor does it print local times::
 
-    >>>> np.datetime64('2000-01-01T00:00:00')
+    >>> np.datetime64('2000-01-01T00:00:00')
```

Comment:
This change is fine, but in general we probably do not want to run refguide against outdated release notes, since the example code may no longer be valid.

Reply:
+1. This snuck in before I excluded that directory